sdk/formrecognizer/ai-form-recognizer/test/node/setup.spec.ts

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) Microsoft Corporation
+// Licensed under the MIT license.
+
+/**
+ * This setup file populates a blob container with the files necessary
+ * to train a Form Recognizer model with the training data
+ */
+
+import fs from "fs-extra";
+import path from "path";
+
+import {
+  BlobServiceClient,
+  AnonymousCredential,
+  generateBlobSASQueryParameters,
+  StorageSharedKeyCredential,
+  ContainerSASPermissions,
+  SASProtocol
+} from "@azure/storage-blob";
+import * as recorder from "@azure/test-utils-recorder";
+
+import { setTrainingContainerSasUrl } from "../util/trainingContainer";
+
+import * as dotenv from "dotenv";
+dotenv.config();
+
+if (!recorder.isPlaybackMode()) {
+  const ASSET_PATH = path.join(__dirname, "..", "test-assets", "training");
```

Comment:
would this relative path still valid when running the built output from dist-* folder?

Reply:
Actually, I think it has the opposite problem where it is _only_ valid from the `dist` folder but not from `dist-test`. I guess I should change the path to be relative to CWD rather than use `__dirname`.

Reply:
We have two different ways to run tests, for code coverage most libraries run files the dist-esm. The other way is run the rollup from dist-test.  It seems tricky here...